src/js/themes/base.d.ts

```diff
@@ -77,6 +77,19 @@ type Colors = typeof colors & {
   [x: string]: ColorType;
 };
 
+interface ButtonKindType {
+  background?: BackgroundType;
+  border?: {
+    color?: ColorType;
+  };
+  color?: ColorType;
+  padding?: {
```

Comment:
I'm thinking we should call it `pad` for grommet consistency.

Reply:
I went back and forth on this. My current thinking is that we use `padding` where it is CSS and we use `pad` where it is being used as a grommet prop. Also, since we already had `padding` in theme.button, it seemed like consistency within the component would be helpful. All that to say, if you think `pad` would be better, I'm happy to switch.

Reply:
I'm ok either as long as we make a decision here.
Your statement makes sense to me, but let's keep in mind for consistency that components like DataTable, List, Select, Tab, Table, and even button.size is using `pad` vs `padding`. but I'm OK proceeding with the direction you suggested.